package.json

```diff
@@ -38,6 +40,12 @@
   "devDependencies": {
     "@commitlint/cli": "7.5.1",
     "@commitlint/config-conventional": "7.5.0",
+    "@storybook/addon-actions": "4.1.11",
```

Comment:
Can't that we in the package of application-components?

Reply:
Sure, I'll move it in there.